### PR TITLE
Refactor/tests to use cardano test plans

### PR DESCRIPTION
**Pull request by NabinKawan**
## List of changes

- Aligned test with cardano-test-plans user stories

## Checklist

- [related issue](https://github.com/IntersectMBO/govtool/issues/)
- [x] My changes generate no new warnings
- [x] My code follows the [style guidelines](https://github.com/IntersectMBO/govtool/tree/main/docs/style-guides) of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the [changelog](https://github.com/IntersectMBO/govtool/blob/main/CHANGELOG.md)
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: tests/govtool-frontend/playwright/global-setup.ts [hunk 1]
```diff
@@ -5,6 +5,9 @@ import { loadAmountFromFaucet } from "@services/faucetService";
 import kuberService from "@services/kuberService";
 import walletManager from "lib/walletManager";
 
+const REGISTER_DREP_WALLETS = 9;
+const DREP_WALLETS = 9;
```
Comment (mesudip) [inline tests/govtool-frontend/playwright/global-setup.ts:9]: Rename variables to have `COUNT` somewhere

File: tests/govtool-frontend/playwright/tests/4-proposal-visibility/proposalVisibility.dRep.spec.ts [hunk 2]
```diff
@@ -48,16 +50,50 @@ test.describe("Temporary DReps", async () => {
...
 
     const govActionDetailsPage = await govActionsPage.viewFirstProposal();
     await govActionDetailsPage.vote(faker.lorem.sentence(200));
+
     await govActionsPage.votedTab.click();
     await govActionsPage.viewFirstVotedProposal();
     expect(false, "No vote context displayed").toBe(true);
   });
+
+  test("4I. Should display the recent vote on same sameshot", async ({
```
Comment (mesudip) [inline tests/govtool-frontend/playwright/tests/4-proposal-visibility/proposalVisibility.dRep.spec.ts:69]: Typo

**Review by mesudip — CHANGES_REQUESTED** [review]

File: tests/govtool-frontend/playwright/global-setup.ts (hunk 1 above)
Comment (NabinKawan) [reply tests/govtool-frontend/playwright/global-setup.ts]: Done

File: tests/govtool-frontend/playwright/tests/4-proposal-visibility/proposalVisibility.dRep.spec.ts (hunk 2 above)
Comment (NabinKawan) [reply tests/govtool-frontend/playwright/tests/4-proposal-visibility/proposalVisibility.dRep.spec.ts]: Done

**Review by mesudip — APPROVED** [review]

**Pull request merged**